.github/workflows/autoupdate.yml

```diff
@@ -150,8 +152,8 @@ jobs:
             if ! git diff $DIFF_BRANCH --quiet $FILE_TO_CHECK; then
               echo "There are changes"
               if [ "$PR_EXISTS" -eq 1 ]; then
-                # Check if there was manual commits
-                LAST_AUTHOR=$(git log -1 --pretty=format:'%an')
+                # Check if the last commit of the existing branch was manual
```

Comment:
The bots I know close their old PRs and open new ones. I'm happy to keep doing it this way but we should think about that, it does seem easier to manage.

Reply:
This would increase a lot the total number of PR, no?

Reply:
Your strategy would be:
- try to update
- check if something changed
- check the opened PR with the good title
  - if you find one compare
    - if it changed check who is the author of the last commit
       - if it is the bot, close the PR and open a new one (THIS WOULD BE THE CHANGE)
       - if it is someone else, warn that there are new updates
  - if you do not find an opened PR with the good title, just open a new one (THIS WOULD BE CHANGED)

Reply:
@mvdbeek I can try to implement this strategy if you want. This would indeed prevent failed reopening but this may lead to multiple PR with the same title if the bot fails to close the PR.